tests/e2e/data_persistence.go

```diff
@@ -282,6 +282,119 @@ var _ = ginkgo.Describe("Data Persistence", func() {
 		}
 	})
 
+	// Data Persistence in case of Dynamic Volume Provisioning with XFS filesystem.
+	// Steps
+	//
+	// 1. Create SC with fstype set to "xfs"
+	// 2. Create a PVC which uses SC created in step1.
+	// 3. Create pod and wait for pod to become ready.
+	// 4. Verify volume is attached and volume is mouned inside pod with xfs filesystem.
+	// 5. Create empty file on the volume to verify volume is writable.
```

Comment:
since we are checking for data persistence, you can dump some random IO

Reply:
Added changes to write data in a file and verifying data read is successful.